lib/std/time.zig

```diff
@@ -83,6 +83,12 @@ pub fn localtime(allocator: std.mem.Allocator) LocalTimeError!TimeZone {
 
 /// ISO 8601 compliant date and time representation.
 pub const DateTime = struct {
+    /// nanoseconds, range 0 to 999
```

Comment:
thanks for the update :) Not sure though why we would need separate fields for ms, us and ns - a `u20` nanosecond field [0-999999999] would allow any amount of fractional seconds, down to ns precision, no?

Reply:
I was following the format for the other parts but using just one field for fractions does make more sense thinking again.